### PR TITLE
Replace `generate_static_mpd` to `generate_static_live_mpd`

### PR DESCRIPTION
**Pull request by Korilakkuma**
Correspond to `TODO` comment.

### CODE REVIEW

File: packager/app/retired_flags.cc [hunk 1]
```diff
@@ -35,9 +35,6 @@ DEFINE_int32(
     num_subsegments_per_sidx,
     0,
     "This flag is deprecated. Use --generate_sidx_in_media_segments instead.");
-DEFINE_bool(generate_widevine_pssh,
```
Comment (kqyang) [inline packager/app/retired_flags.cc]: Please don't remove this flag. 

Add the retired flag 'generate_static_mpd' here in a similar way to 'generate_widevine_pssh' here.

Comment (Korilakkuma) [reply packager/app/retired_flags.cc]: Sorry ... I mistaken ... :bow:

File: packager/app/retired_flags.cc [hunk 2]
```diff
@@ -89,6 +92,16 @@ bool InformRetiredPsshGenerationFlag(const char* flagname, bool value) {
   return true;
 }
 
+bool InformRetiredGenerateStaticMpdFlag(const char* flagname, bool value) {
+  if (value) {
+    fprintf(stderr,
+            "WARNING: %s is deprecated and ignored. Please switch to "
+            "generate_static_live_mpd.\n",
```
Comment (kqyang) [inline packager/app/retired_flags.cc]: can you add "--" before generate_static_live_mpd?

**Comment by kqyang** [comment]
Sorry for not noticing it earlier. Please update the commands in the docs too, e.g. in https://github.com/google/shaka-packager/blob/master/docs/source/tutorials/dash.rst. Please do a source code search and replace all matchings.

**Comment by kqyang** [comment]
Thanks for the update. I believe there are others: https://github.com/google/shaka-packager/search?q=--generate_static_mpd&unscoped_q=--generate_static_mpd. Can you the ones in other *.rst too? Thanks.

**Comment by kqyang** [comment]
@Korilakkuma The pull request has been merged. Thanks a lot for your help!

**Pull request merged**